internal/sirius/get_events.go

```diff
@@ -0,0 +1,15 @@
+package sirius
+
+import (
+	"fmt"
+)
+
+func (c *Client) GetEvents(ctx Context, id int, caseId int) (any, error) {
```

Comment:
This should probably also have a contract test. Appreciate we're not doing much with the structure yet, so it's hard to add a contract to the body, but we could at least say "it must return an array of objects".